rally_ovs/plugins/ovs/scenarios/ovn_network.py

```diff
@@ -15,12 +15,105 @@
 from rally.common import logging
 from rally_ovs.plugins.ovs.scenarios import ovn
 
+from rally import exceptions
 from rally.task import scenario
 from rally.task import validation
 
 LOG = logging.getLogger(__name__)
 
+class LogicalNetwork():
+    def __init__(self):
+        self.sandboxes = []
+        self.ports_per_network = 0
 
```

Comment:
better to add 'self.lswitch = None' in __init__(), otherwise get_lswitch() before set_lswitch() may raise a exception